pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,49 +13,34 @@ node('jenkins-master') {
 }
 
 notify.wrap {
-  def retries = 3
   def repo = 'webserv/imgserv'
   def tag = 'dax_latest'
 
-  def run = {
-    dir('imgserv/lsst-dm-ci') {
+  node('docker') {
+    timeout(time: 1, unit: 'HOURS') {
+        git([
+          url: util.githubSlugToUrl('lsst/dax_imgserv'),
+          branch: 'master'
+        ])
+
         stage('build') {
-            retry(retries) {
-                build(repo)
-            }
+            withEnv(["DOCKER_REPO=$repo"]) {
+                util.bash './lsst-dm-ci/prod_image.sh'
+             }
```

Comment:
The indenting here appears to have an odd number of spaces.

Reply:
Fixed.